be/src/fs/fs_util.h

```diff
@@ -197,9 +198,10 @@ inline bool starts_with(std::string_view s, std::string_view prefix) {
 }
 
 inline bool is_s3_uri(std::string_view uri) {
-    return starts_with(uri, "oss://") || starts_with(uri, "s3n://") || starts_with(uri, "s3a://") ||
-           starts_with(uri, "s3://") || starts_with(uri, "cos://") || starts_with(uri, "cosn://") ||
-           starts_with(uri, "obs://") || starts_with(uri, "ks3://") || starts_with(uri, "tos://");
+    return starts_with(uri, "s3n://") || starts_with(uri, "s3a://") || starts_with(uri, "s3://") ||
+           (config::object_storage_use_aws_sdk_default &&
```

Comment:
This configuration is a bit tricky and not very extensible, can we add some kind of mechanism that allows to specify the filesystem for a certain scheme through a configuration file? like this:
```
hdfs_scheme=obs,ks3
s3_scheme=oss,cos
```

Reply:
cc @kevincai

Reply:
in general case, these object storage should be compatible with s3, but in some case vendor may add some optimization that is not compatible with s3, in this case user can't use aws sdk, and use the hadoop jar that vendor supplied. what i mean is even same schema, some case should use aws sdk and some case not.

Reply:
> even same schema, some case should use aws sdk and some case not

It looks like that your current patch doesn't solve the problem?

Reply:
this is a global configure, it means that if the configure is turned on, all the related schema will be bypass s3 sdk and fallback to hdfs jar. what if they uses both tos and ks3 in the same cluster, and one is expecting to use s3 sdk and the other uses hdfs jar? can we do something to the broker job to help selecting the underlying fs?